src/libraries/Microsoft.Extensions.DependencyModel/src/RuntimeAssetGroup.cs

```diff
@@ -8,8 +8,8 @@ namespace Microsoft.Extensions.DependencyModel
 {
     public class RuntimeAssetGroup
     {
-        private IReadOnlyList<string> _assetPaths;
-        private IReadOnlyList<RuntimeFile> _runtimeFiles;
+        private IReadOnlyList<string>? _assetPaths;
+        private IReadOnlyList<RuntimeFile>? _runtimeFiles;
 
         public RuntimeAssetGroup(string runtime, params string[] assetPaths) : this(runtime, (IEnumerable<string>)assetPaths) { }
```

Comment:
It looks like `string runtime` should be `string? runtime`. Nothing is checking it for `null`.